server/game/effects.js

```diff
@@ -136,14 +136,15 @@ const Effects = {
             }
         };
     },
-    setSetupGold: function(value) {
```

Comment:
Did this function get deleted intentionally? Or moved? I might just be missing it, but I don't see it in the rest of the file changes.

Reply:
That is just a remainder from the throneteki code which used setup gold. I noticed it while working on this and removed.